docs/reference/rollup/overview.asciidoc

```diff
@@ -1,7 +1,10 @@
 [role="xpack"]
 [testenv="basic"]
 [[rollup-overview]]
-== Overview
+=== {rollup-cap} overview
+++++
```

Comment:
Structurally, I think it would be better if we could incorporate the "overview" material into the top level section landing pages, instead of just making them link farms. Or make the top-level headings just landmarks and not actually navigable links in the TOC (as @gchaps suggested). Then this content would have the "overview" keyword for SEO, but we'd get rid of the extra click to get to meaningful content.) For now, this is consistent with other topics.

Reply:
I agree that this will be worth discussing in more detail as we roll out the reorg and strive for consistency, but for now I'll leave as-is.